gotocompany/compass/v1beta1/service.proto

```diff
@@ -1198,6 +1212,12 @@ message CreateAssetProbeResponse {
   string id = 1;
 }
 
+message SyncAssetsRequest {
+  repeated string index_names = 1;
```

Comment:
Let's use this instead?
https://github.com/goto/proton/blob/34aa07ae5769b44043beb3e98da7d927904ac557/gotocompany/compass/v1beta1/service.proto#L974-L985

Reply:
updated, considering to filter by services for simplifying things